--- "F:/Projects/v11 threads og/v11 threads/v11 threads/map-app/src/components/Sidebar.jsx"	2026-03-31 15:42:18.310490400 +0500
+++ "F:/Projects/v11 threads with MCA/v11 threads/map-app/src/components/Sidebar.jsx"	2026-06-02 21:43:37.603097400 +0500
@@ -1,6 +1,7 @@
 import React, { Suspense, lazy, useMemo, useState } from 'react';
 import toast from 'react-hot-toast';
 import { useMapStore } from '../store/useMapStore';
+import McaPanel from './McaPanel';
 
 const RouteStatsPanel = lazy(() => import('./RouteStatsPanel'));
 
@@ -535,11 +536,13 @@
           </div>
         ))}
 
-        {renderDesktopAccordionSection('tools', 'Многокритериальный анализ', renderAnalysisContent())}
 
         {store.routeStats &&
           renderDesktopAccordionSection('stats', 'Статистика маршрута', renderRouteStats())}
 
+        {/* ### КОМПОНЕНТ МИКРОСЕРВИСА ### */}
+        {renderDesktopAccordionSection('mca', 'Анализ пригодности территории', <McaPanel />)}
+        
         {store.user &&
           renderDesktopAccordionSection('myReviews', 'Мои отзывы', renderReviewsContent())}
 
